sdk/storage/azure-storage-blob-v2/src/main/java/com/azure/storage/blob/AzureBlobStorageBuilder.java

```diff
@@ -0,0 +1,308 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.storage.blob;
```

Comment:
Maybe we should have these use v2 in the package as well to avoid any name collisions (e.g. when testing side by side).